discord-ptb.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+FLATPAK_ID=${FLATPAK_ID:-"com.discordapp.DiscordPTB"}
+# RPC was discontinued in DiscordPTB
```

Comment:
Out of curiosity is there somewhere that this was publicized? I've been using the RPC protocol on IPC and Websockets and would like to know if/when discord plans to fully kill it.

Thanks in advanced!

Reply:
Honestly, this was something I did a while ago so I don't remember from where I got the info (probably some forum) but if I remember correctly, ptb didn't seem to Open rpc (even though I tried) and I also https://discord.com/developers/docs/rich-presence/best-practices deprecates the standard